app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -429,7 +429,6 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
             // experiment on features close to startup.
             // But we need viaduct (the RustHttp client) to be ready before we do.
             components.analytics.experiments.apply {
-                initialize()
```

Comment:
@jhugman is this change still needed? I don't remember we removed it because debugging.

Reply:
No, we do not need this any more because we not have the new `initialize` method that does all the things inside it.

Reply:
Thanks for the clarification 👍🏽